providers/os/resources/smbios/win.go

```diff
@@ -112,11 +116,11 @@ func (s *WindowsSmbiosManager) Info() (*SmBiosInfo, error) {
 			SerialNumber: winBios.BaseBoard.SerialNumber,
 			Version:      winBios.BaseBoard.Version,
 		},
-		ChassisInfo: ChassisInfo{
-			Vendor:       winBios.Chassis.Manufacturer,
-			Model:        toString(winBios.Chassis.Model),
-			Version:      winBios.Chassis.Version,
-			SerialNumber: winBios.Chassis.SerialNumber,
+		ChassisInfo: ChassisInfo{ // TODO: Might want to make this a slice
```

Comment:
can this be nil? if `winBios.Chassis` is 0, we may as well jst set this to nil. we're currently filling in an empty struct that holds no value. if this cannot be nil, then thats ok